docker/Dockerfile

```diff
@@ -400,36 +404,15 @@ RUN install-from-source https://github.com/nanopb/nanopb/archive/refs/tags/0.4.9
 ### ADD NEW PROGRAMS/LIBRARIES HERE ###
 #######################################
 
-# Install python libraries
-RUN pip install \
-    pygit2==1.13.3 \
-    xxhash \
-    pylint \
-    termcolor \
-    stringcase \
-    Pillow \
-    protobuf==4.21.2 \
-    mycroft-mimic3-tts \
```

Comment:
`mycroft-mimic3-tts` probably needs to stay for now because it's called from C++ using a `system` call and will most likely bypass the poetry venv

Reply:
We should replace this with the piper C++ library `https://github.com/rhasspy/piper` but can be done in a later PR

Reply:
It all works with the venv stuff, when you run anything via ./b it's all set up in a poetry run command so the paths all work

Reply:
Okay cool